flang/include/flang/Lower/AbstractConverter.h

```diff
@@ -357,6 +358,9 @@ class AbstractConverter {
   /// functions in order to be in sync).
   virtual mlir::SymbolTable *getMLIRSymbolTable() = 0;
 
+  virtual Fortran::lower::AccRoutineInfoMappingList &
```

Comment:
This formatting seems weird.  If it's not from `clang-format`, please run it though a formatter.